tailscale/rootfs/etc/s6-overlay/s6-rc.d/tailscaled/run

```diff
@@ -14,6 +15,13 @@ options+=(--statedir=/data/state)
 options+=(--no-logs-no-support)
 options+=(--tun=userspace-networking)
 
+# If outbound proxy port is specified, enable outbound proxy
+outbound_proxy_port=$(bashio::addon.port 1080)
+if bashio::var.has_value "${outbound_proxy_port}"; then
+  options+=(--socks5-server=0.0.0.0:${outbound_proxy_port})
```

Comment:
This add-on runs on the host network, and setting it to 0.0.0.0 will try to bind it on every interface it has available. I would recommend not doing that, as a conflict with the many many (virtual) interfaces, is hard to avoid.

Reply:
Yes, and when you can merge the "drop userspace networking" branch, it will be useless.

Should I close this PR?